samples/client/petstore/java/jersey2-java6/docs/MixedPropertiesAndAdditionalPropertiesClass.md

```diff
@@ -4,7 +4,7 @@
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**uuid** | **String** |  |  [optional]
+**uuid** | [**UUID**](UUID.md) |  |  [optional]
```

Comment:
@ePaul there's a discussion about updating `isPrimitiveType` to include UUID as well. IMO, `isPrimitiveType` should include UUID.

(the issue was caused by the change that maps `uuid` as UUID instead of String in Java)